deploy/runtime/versions.lk

```diff
@@ -902,10 +789,13 @@ version_upgrade{'2015.11.16'} = define() {
 
 version_upgrade{'2015.10.29'} = define() {
 	const cfg = config();
-	if( cfg.tech == 'Flat Plate PV' && (cfg.fin == 'Residential' || cfg.fin == 'Commercial' ) && value('en_batt') == 1)
+	if( ( cfg.tech == 'Flat Plate PV' && value('en_batt') == 1 ) || cfg.tech == 'PV Battery') // in case config was changed from flat plate pv to pv battery in 2020.2.29 function
 	{
-		x = oldvalue('batt_replacement_cost');
-		value( 'batt_replacement_cost', x[0] ); // batt_replacement_cost is now single value
+		if ( cfg.fin == 'Residential' || cfg.fin == 'Commercial' ) 
+		{
+			x = oldvalue('batt_replacement_cost');
+			value( 'batt_replacement_cost', x[0] ); // batt_replacement_cost is now single value
```

Comment:
It looks like we reversed ourselves on this between 2020.2.29 and 2020.11.29 - the battery replacement cost is an array again. I assume we should leave this as is for the intermediate functions?

Reply:
My understanding is that batt_replacement_cost has always been an array
![image](https://user-images.githubusercontent.com/6498311/107140059-e2143100-68dc-11eb-9fb9-aadd6c666f06.png)

I cannot find 'batt_replacement_cost' in any UI forms on any compute modules - the variable used is 'om_replacement_cost1':
![image](https://user-images.githubusercontent.com/6498311/107140172-88603680-68dd-11eb-83ce-e8aa834afe96.png)

Reply:
Ah right, I was thinking of batt_cycle_cost which I changed in the latest version. It looks like the transition from batt_replacement_cost to om_replacement_cost1 happened in the upgrade to 2020.2.29.

Reply:
batt_replacement_cost was originally a number on the Battery page, then we changed it to an array and moved it to the O&M page. Here's a screenshot from SAM 2016.3.14:

![image](https://user-images.githubusercontent.com/27711933/107587827-54c02d80-6bb7-11eb-8e94-e9ddcf9ced1b.png)